dtfs-central-api/src/v1/routes/user-routes.js

```diff
@@ -66,4 +66,9 @@ userRouter.route('/:id')
     getUserController.findOneUserGet,
   );
 
+userRouter.route('/')
+  .get(
+    getUserController.list,
```

Comment:
I can't see where this route is being used?

Reply:
used in api-test-users.js file